skeletons/web-extension/boot.js

```diff
@@ -0,0 +1,16 @@
+// This script runs at document_start, avoid adding more stuff here!
+
+// TODO: make this conditional, only do it when requested.
+
+let id = `ember-inspector-boot-${(Math.random() * 100000000).toFixed(0)}`;
+
+let script = document.createElement('script');
```

Comment:
I'm curious if these `let`'s pollute "global space" for the consumer, have you tested?

Reply:
It does not, it's in its own special scope (which is why I can't just set `window.EmberENV = { _DEBUG_RENDER_TREE: true };` here in the first place)